src/shared/ui/fullPageWrapper/full-page-wrapper.module.css

```diff
@@ -0,0 +1,3 @@
+.wrapper {
+    min-height: 22rem;
```

Comment:
please don't use rem for page size
rem depends on base font size and it will work unpredictable on mobile devices

Reply:
Its just a spacer for situations where I need to use Spinner component. Just a decoration. Nothing more.

Reply:
then you can use viewport

Reply:
Nope, if I use spinner in topNews and newsFeed separately)